installer/Controller/Upgrader/Traits/UpgraderUtilityTrait.php

```diff
@@ -19,47 +19,50 @@
 
 namespace OrangeHRM\Installer\Controller\Upgrader\Traits;
 
-use mysqli;
+use Doctrine\DBAL\Schema\AbstractSchemaManager;
+use Exception;
+use OrangeHRM\Installer\Util\Connection;
 
 trait UpgraderUtilityTrait
```

Comment:
I think this should be a class, not a trait

Reply:
created a utility class